block-layer/codecs/DAG-JSON.md

```diff
@@ -14,14 +14,56 @@ Contrary to popular belief, JSON as a format supports Big Integers. It's only
 JavaScript itself that has trouble with them. This means JS implementations
 of `dag-json` can't use the native JSON parser and serializer.
 
+### Version 0
+
+This is an old version of `dag-json` that reserved the `"/"` key in order to
+represent binary and link data types.
+
 #### Binary Type
 
 ```javascript
 {"/": { "base64": String }}
 ```
 
-### Link Type
+#### Link Type
 
 ```javascript
 {"/": String /* base encoded CID */}
-```
\ No newline at end of file
+```
+
+### Version 1
+
+#### Format
+
+The internal data format is valid JSON but is **NOT** identical to the decoded
+node data codecs produce.
+
+Example internal format:
+
+```javascript
+{ "/": { "_": 1 },
```

Comment:
Wouldn't it be simpler to just add a field called `version` and move `meta` and `data` to the top-level?

Reply:
“meta” and “data” *are* top level. using “_” was just to save space but we can do whatever we want.

Reply:
Oh, sorry I missed that they are top-level.